Formula/prs.rb

```diff
@@ -0,0 +1,33 @@
+class Prs < Formula
```

Comment:
Can we use `PrsCli` instead of `Prs`?
```suggestion
class PrsCli < Formula
```

Reply:
The binary and the project is called `prs`. It is only referenced to as `prs-cli` in source code because Rust workspaces are used.

I therefore think it's better to leave as is?

Reply:
> The binary and the project is called `prs`. It is only referenced to as `prs-cli` in source code because Rust workspaces are used.

Unfortunately, there is no golden rule on the formula name mapping with the artifact name. Even though it is desired mainly because it is easy to remember (which i also doubt, because `prs-cli` is equally easy to remember), the other concern would be preventing some other formulae using this same name (which would be some library using this name, which is also unlikely)

Yeah, I am on the fence of whether using prs-cli or not, but I think it is CLI only, right? Then why not using a name to manifest it. What's more in the version output, I did see `prs-cli x.x.x`.

Does it make better sense?

Reply:
> Yeah, I am on the fence of whether using prs-cli or not, but I think it is CLI only, right?

It is not. There's a GUI front-end amongst other things as well. For brew the package currently only provides a CLI, but more will be added once it stabilizes for macOS.

Other package systems use `prs` as well, for the package name. It would be great if this could be kept consistent.

The `-cli` suffix in help is actually there by mistake which propagated automatically when reorganizing the project. This is now removed, but this change hasn't seen a release yet.

Reply:
fair enough. Yeah, I think I can take it.